svc-telemetry/telemetry/telemetry.go

```diff
@@ -375,15 +315,7 @@ func (e *ExternalInterface) GetTrigger(req *teleproto.TelemetryRequest) response
 // UpdateTrigger ...
 func (e *ExternalInterface) UpdateTrigger(req *teleproto.TelemetryRequest) response.RPC {
 	var resp response.RPC
-	resp.Header = map[string]string{
-		"Allow":             `"GET"`,
-		"Cache-Control":     "no-cache",
-		"Connection":        "keep-alive",
-		"Content-type":      "application/json; charset=utf-8",
-		"Transfer-Encoding": "chunked",
-		"OData-Version":     "4.0",
-	}
-
+	resp.Header = header
```

Comment:
common header defined has only "GET" as value for "Allow", for triggers PATCH also should be included.

Reply:
addressed